packages/components/src/internal/components/picklist/PicklistEditModal.tsx

```diff
@@ -119,7 +119,6 @@ export const PicklistEditModal: FC<Props> = memo(props => {
                         Category: shared ? PUBLIC_PICKLIST_CATEGORY : PRIVATE_PICKLIST_CATEGORY,
                     })
                 );
-                incrementClientSideMetricCount(metricFeatureArea, 'updatePicklist');
```

Comment:
was this metric call being done somewhere else, or was there a different reason for removing it?